src/store/mod.rs

```diff
@@ -38,6 +38,22 @@ mod writer;
 pub use self::reader::StoreReader;
 pub use self::writer::StoreWriter;
 
+trait Compression {
+    fn compress(uncompressed: &[u8], compress: &mut Vec<u8>);
+    fn decompress(compressed: &[u8], decompressed: &mut Vec<u8>);
+}
```

Comment:
Is this to allow an eventual "plug and play"? I didn't see it being used.

Reply:
Removing this.